web_poet/pages.py

```diff
@@ -81,10 +82,9 @@ def _to_item(self, *args, **kwargs):
     return _to_item  # type: ignore[return-value]
 
 
-class ItemPage(Injectable, Returns[ItemT]):
-    """Base Page Object, with a default :meth:`to_item` implementation
-    which supports web-poet fields.
-    """
+class Extractor(Returns[ItemT], FieldsMixin):
+    """The base class for page objects and other extractors, providing support
+    for fields."""
```

Comment:
To make it clear that `ItemPage` is the base class for page objects, maybe it would be best not to mention page objects here.

```suggestion
    """Base class for field support."""
```